config-linux.md

```diff
@@ -189,6 +189,42 @@ In addition to any devices configured with this setting, the runtime MUST also s
 * [`/dev/ptmx`][pts.4].
   A [bind-mount or symlink of the container's `/dev/pts/ptmx`][devpts].
 
+## <a name="configLinuxNetworkDevices" />Network Devices
+
+Linux network devices are entities that send and receive data packets.
+They are not represented as files in the /dev directory, unlike block devices, network devices are represented with the [`net_device`][net_device] data structure in the Linux kernel.
+Network devices can belong to only one network namespace and use a set of operations distinct from regular file operations. Examples of network devices include Ethernet cards, loopback devices, and virtual devices like bridges, VLANs, and MACVLANs.
+
+This schema focuses solely on moving existing network devices identified by name from the host network namespace into the container network namespace. It does not cover the complexities of network device creation or network configuration, such as IP address assignment, routing, and DNS setup.
+
+**`netDevices`** (object, OPTIONAL) set of network devices that MUST be made available in the container. The runtime is responsible for providing these devices; the underlying mechanism is implementation-defined.
```

Comment:
I think here is array of objects, and each object has a string field with name `name`.

You want an array at some point, for the net devices, but here you say object and later you say string. I think here you want to say array of object, like here: https://github.com/opencontainers/runtime-spec/pull/1271/files#diff-048d23d864e15683f516d2c1768965d546e87f8a59b2606cf2f2d52500ba5a32R127

OHH, you want two names, the host network interface and the name to assign in the container, right? Maybe you want array of objects and each object has two fields, the name of the interface of the host and the name to assign to the container interface?

Reply:
If you see the previous proposal, there were multiple attributes before , leaving this as an object allows to extend the API in a backwards compatible way
```
    "linux": {
        "netDevices": {
            "eth0": {
                "name": "container_eth0",
                 "new_field_here": "asdasd",
            },
            "ens4": {},
            "ens5": {}
        }
    }
```

I still think the object to represent a dictionary is better than an array because avoids duplicate names on the configuration that can cause runtime errors https://github.com/opencontainers/runtime-spec/pull/1271#issuecomment-2655058164

Reply:
Oh, sorry, I missed this was a "set" ("set of network devices ...")